<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vvMainWindow</class>
 <widget class="QMainWindow" name="vvMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="vvResources.qrc">
    <normaloff>:/vvResources/Icons/logo.png</normaloff>:/vvResources/Icons/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="actionOpen_Sensor_Stream"/>
    <addaction name="actionChoose_Calibration_File"/>
    <addaction name="separator"/>
    <addaction name="actionExport_To_KiwiViewer"/>
    <addaction name="actionSave_CSV"/>
    <addaction name="actionSave_Screenshot"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSpreadsheet"/>
    <addaction name="actionMeasurement_Grid"/>
    <addaction name="actionGrid_Properties"/>
    <addaction name="actionChoose_Calibration_File"/>
    <addaction name="separator"/>
    <addaction name="actionPlay"/>
    <addaction name="actionRecord"/>
    <addaction name="separator"/>
    <addaction name="actionPython_Console"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Basic Controls</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="actionOpen_Sensor_Stream"/>
   <addaction name="actionChoose_Calibration_File"/>
   <addaction name="separator"/>
   <addaction name="actionReset_Camera"/>
   <addaction name="actionSelect_All_Points"/>
   <addaction name="actionSpreadsheet"/>
   <addaction name="actionPython_Console"/>
  </widget>
  <widget class="pqColorToolbar" name="colorToolBar">
   <property name="windowTitle">
    <string>Color Controls</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="playbackToolbar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionGo_To_Start"/>
   <addaction name="actionSeek_Backward"/>
   <addaction name="actionPlay"/>
   <addaction name="actionSeek_Forward"/>
   <addaction name="actionGo_To_End"/>
   <addaction name="actionLoop"/>
   <addaction name="actionRecord"/>
  </widget>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="statusTip">
    <string>Exit Application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/WiresharkDoc-128.png</normaloff>:/VelodyneHDLPlugin/WiresharkDoc-128.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open File</string>
   </property>
   <property name="statusTip">
    <string>Open Data File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSelect_Visible_Points">
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/pqSurfaceSelectionPoint24.png</normaloff>:/vvResources/Icons/pqSurfaceSelectionPoint24.png</iconset>
   </property>
   <property name="text">
    <string>Select Visible Points</string>
   </property>
   <property name="toolTip">
    <string>Select Visible Points (s)</string>
   </property>
  </action>
  <action name="actionSelect_All_Points">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/pqFrustumSelectionPoint24.png</normaloff>:/vvResources/Icons/pqFrustumSelectionPoint24.png</iconset>
   </property>
   <property name="text">
    <string>Select All Points</string>
   </property>
   <property name="toolTip">
    <string>Select All Points</string>
   </property>
  </action>
  <action name="actionSpreadsheet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/pqSpreadsheet16.png</normaloff>:/vvResources/Icons/pqSpreadsheet16.png</iconset>
   </property>
   <property name="text">
    <string>Spreadsheet</string>
   </property>
   <property name="toolTip">
    <string>Spreadsheet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionOpen_Sensor_Stream">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/velodyne_hdl.png</normaloff>:/VelodyneHDLPlugin/velodyne_hdl.png</iconset>
   </property>
   <property name="text">
    <string>Open Sensor Stream</string>
   </property>
  </action>
  <action name="actionChoose_Calibration_File">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/angle_icon.png</normaloff>:/VelodyneHDLPlugin/angle_icon.png</iconset>
   </property>
   <property name="text">
    <string>Choose Calibration File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionSeek_Backward">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/media-seek-backward.png</normaloff>:/VelodyneHDLPlugin/media-seek-backward.png</iconset>
   </property>
   <property name="text">
    <string>Seek Backward</string>
   </property>
   <property name="toolTip">
    <string>Seek Backward</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/media-playback-start.png</normaloff>:/VelodyneHDLPlugin/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSeek_Forward">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/media-seek-forward.png</normaloff>:/VelodyneHDLPlugin/media-seek-forward.png</iconset>
   </property>
   <property name="text">
    <string>Seek Forward</string>
   </property>
   <property name="toolTip">
    <string>Seek Forward</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/media-playback-pause.png</normaloff>:/VelodyneHDLPlugin/media-playback-pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/media-record.png</normaloff>:/VelodyneHDLPlugin/media-record.png</iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionPython_Console">
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/python_logo_124.png</normaloff>:/vvResources/Icons/python_logo_124.png</iconset>
   </property>
   <property name="text">
    <string>Python Console</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionReset_Camera">
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/pqResetCamera32.png</normaloff>:/vvResources/Icons/pqResetCamera32.png</iconset>
   </property>
   <property name="text">
    <string>Reset Camera</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close Data</string>
   </property>
   <property name="toolTip">
    <string>Close Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionMeasurement_Grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Measurement Grid</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionGo_To_Start">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/media-skip-backward.png</normaloff>:/VelodyneHDLPlugin/media-skip-backward.png</iconset>
   </property>
   <property name="text">
    <string>Go To Start</string>
   </property>
  </action>
  <action name="actionGo_To_End">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/media-skip-forward.png</normaloff>:/VelodyneHDLPlugin/media-skip-forward.png</iconset>
   </property>
   <property name="text">
    <string>Go To End</string>
   </property>
  </action>
  <action name="actionSave_Screenshot">
   <property name="text">
    <string>Save Screenshot</string>
   </property>
  </action>
  <action name="actionSave_CSV">
   <property name="text">
    <string>Save CSV</string>
   </property>
  </action>
  <action name="actionLoop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/media-loop.png</normaloff>:/VelodyneHDLPlugin/media-loop.png</iconset>
   </property>
   <property name="text">
    <string>Loop</string>
   </property>
  </action>
  <action name="actionExport_To_KiwiViewer">
   <property name="icon">
    <iconset resource="VelodyneHDL/images/resources.qrc">
     <normaloff>:/VelodyneHDLPlugin/kiwi_200.png</normaloff>:/VelodyneHDLPlugin/kiwi_200.png</iconset>
   </property>
   <property name="text">
    <string>Export To KiwiViewer</string>
   </property>
   <property name="toolTip">
    <string>Export To KiwiViewer</string>
   </property>
  </action>
  <action name="actionGrid_Properties">
   <property name="text">
    <string>Grid Properties</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqColorToolbar</class>
   <extends>QToolBar</extends>
   <header>pqColorToolbar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="vvResources.qrc"/>
  <include location="VelodyneHDL/images/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered()</signal>
   <receiver>vvMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
